cmd/sos.go

```diff
@@ -55,6 +55,15 @@ var sosCmd = &cobra.Command{
 	Short:            "Simple Object Storage management",
 	Long:             sosCmdLongHelp(),
 	TraverseChildren: true,
+
+	Hidden: true,
+	PersistentPreRun: func(_ *cobra.Command, _ []string) {
+		fmt.Fprintln(os.Stderr,
+			`**********************************************************************
+The "exo sos" commands are deprecated and replaced by "exo storage",
```

Comment:
I don't get it ? https://forum.internal.exoscale.ch/t/cli-new-commands-exo-storage/371/8

Already fighting a long legacy of inconsistency why take this route here ? #onvajamaisyarriver